src/pieces/commandManager.ts

```diff
@@ -85,6 +85,10 @@ async function runCommand(msg: Message) {
 
 	const command = getCommand(msg.client, commandName);
 	if (!command || command.enabled === false) return;
+	if (msg.channel.id !== CHANNELS.SAGE && command.restricted) {
+		msg.author.send('This command is only available in #sages_place.');
```

Comment:
This message should probably self destruct after a few seconds.

Reply:
Additionally, this could be an ephemeral once everything is a slash command

Reply:
I didn't realize this was an author send. You can safely ignore the above suggestion.